src/selectivity-templates.js

```diff
@@ -267,6 +285,22 @@ Selectivity.Templates = {
                 escape(options.text) +
             '</span>'
         );
+    },
+
+    /**
+     * Renders the selected item in compliance <select> element as <option>.
+     *
+     * @param options Options object containing the following properties
+     *                id - Identifier for the item.
+     *                removable - Boolean whether a remove icon should be displayed.
```

Comment:
Deleted. I've kept it as this is the same list of options as the other template uses